shared/actions/chat.js

```diff
@@ -375,6 +381,24 @@ function * _startConversation (action: StartConversation): SagaGenerator<any, an
   }
 }
 
+function * _openFolder (): SagaGenerator<any, any> {
+  const selectedSelector = (state: TypedState) => state.chat.get('selectedConversation')
+  const conversationIDKey = yield select(selectedSelector)
+
+  const inboxSelector = (state: TypedState) => {
+    return state.chat.get('inbox').find(convo => convo.get('conversationIDKey') === conversationIDKey)
+  }
+
+  const inbox = yield select(inboxSelector)
+  if (inbox) {
+    const helper = inbox.get('info').visibility === CommonTLFVisibility.public ? publicFolderWithUsers : privateFolderWithUsers
```

Comment:
Known core bug where this comes in as 'any'. I compare vs public so that'll be treated as private